src/connectors/refinement-list/__tests__/connectRefinementList-test.js

```diff
@@ -711,6 +711,86 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/refinement-
     ]);
   });
 
+  it('show more should toggle between two limits after search', () => {
```

Comment:
There's possibly a better way to test this.

I rely on the number of renders and the fact that `toggleShowMore` is defined after having searched for items (this test fails when run with the previous `connectRefinementList`).